src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_inventory.html.twig

```diff
@@ -3,7 +3,11 @@
     <div class="ui attached segment">
         {% if product.simple %}
             {{ form_row(form.variant.onHand) }}
+            <br />
+
             {{ form_row(form.variant.tracked) }}
+            <div style="color: rgba(0, 0, 0, .6)">{{ form_help(form.variant.tracked) }}</div>
```

Comment:
IMHO inline styles should be discouraged. Can you add a new style to src/Sylius/Bundle/UiBundle/Resources/private/sass/main.scss?